src/app/modules/main/profile_section/contacts/view.nim

```diff
@@ -108,6 +108,10 @@ QtObject:
   proc removeTrustStatus*(self: View, publicKey: string) {.slot.} =
     self.delegate.removeTrustStatus(publicKey)
 
+  proc trustStatusRemoved*(self: View, publicKey: string) {.signal.}
```

Comment:
We should pass the new status to know which one was removed (trusted or untrusted)

Reply:
Yeah but I can only get the new status from the signal which is not very interesting here; I'd need to know the old one